### PR TITLE
kill process when interrupting twice

### LINKED ISSUE #31: kill process when hitting `ctrl+c` twice
Recently we introduced gracefully shutdown behaviour when listening on signals. This sucks while developing, because one needs to wait for the shutdown. I would be cool to kill the process when hitting `ctrl+c` twice.

### PR DESCRIPTION
**Pull request by zyndiecate**
fixes #31

### CODE REVIEW

**Comment by zyndiecate** [comment]
ping @ZeissS

File: server.go [hunk 1]
```diff
@@ -59,6 +59,8 @@ type Server struct {
 
 	ctxConstructor CtxConstructor
 
+	signal             os.Signal
```
Comment (ZeissS) [inline server.go]: this var makes no sense.

File: server.go [hunk 2]
```diff
@@ -59,6 +59,8 @@ type Server struct {
...
+	signalCounter      int
```
Comment (ZeissS) [inline server.go]: Can you use an atomic counter for this?

File: server.go (hunk 1 above)
Comment (zyndiecate) [inline server.go]: removed

File: server.go (hunk 2 above)
Comment (zyndiecate) [inline server.go]: done

**Comment by ZeissS** [comment]
LGTM

**Pull request merged**